pkg/ddevapp/config.go

```diff
@@ -420,3 +453,60 @@ func prepLocalSiteDirs(base string) error {
 
 	return nil
 }
+
+// validateCommandYaml validates command hooks and tasks defined in extend-commands for config.yaml
```

Comment:
Another failure (panic) with ddev import-db (windows):
```
PS C:\Users\randy\workspace\randyfay> ddev import-db --src=randyfay.sql.gz
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x7ddba8]

goroutine 1 [running]:
github.com/drud/ddev/pkg/plugins/platform.(*LocalApp).GetName(0xc0422c8050, 0x0, 0xcca6c0)
        /go/src/github.com/drud/ddev/pkg/plugins/platform/local.go:151 +0x8
github.com/drud/ddev/cmd/ddev/cmd.glob..func7(0xcff740, 0xc04211cf40, 0x0, 0x1)
        /go/src/github.com/drud/ddev/cmd/ddev/cmd/import-db.go:43 +0x31e
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).execute(0xcff740, 0xc04211cf20, 0x1, 0x1, 0xcff740, 0xc042
11cf20)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:603 +0x232
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xd00400, 0x0, 0xd00400, 0xc0420b7f50)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:689 +0x340
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).Execute(0xd00400, 0xc0420b7f68, 0x8dd648)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:648 +0x32
github.com/drud/ddev/cmd/ddev/cmd.Execute()
        /go/src/github.com/drud/ddev/cmd/ddev/cmd/root.go:98 +0x42
main.main()
        /go/src/github.com/drud/ddev/cmd/ddev/main.go:6 +0x27
PS C:\Users\randy\workspace\randyfay> ddev import-db --src=randyfay.sql.gz
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x7ddba8]

goroutine 1 [running]:
github.com/drud/ddev/pkg/plugins/platform.(*LocalApp).GetName(0xc0420501b0, 0x0, 0xcca6c0)
        /go/src/github.com/drud/ddev/pkg/plugins/platform/local.go:151 +0x8
github.com/drud/ddev/cmd/ddev/cmd.glob..func7(0xcff740, 0xc04211f030, 0x0, 0x1)
        /go/src/github.com/drud/ddev/cmd/ddev/cmd/import-db.go:43 +0x31e
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).execute(0xcff740, 0xc04211f010, 0x1, 0x1, 0xcff740, 0xc042
11f010)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:603 +0x232
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xd00400, 0x0, 0xd00400, 0xc0420b7f50)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:689 +0x340
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).Execute(0xd00400, 0xc0420b7f68, 0x8dd648)
        /go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:648 +0x32
github.com/drud/ddev/cmd/ddev/cmd.Execute()
        /go/src/github.com/drud/ddev/cmd/ddev/cmd/root.go:98 +0x42
main.main()
        /go/src/github.com/drud/ddev/cmd/ddev/main.go:6 +0x27
```

This is with this stanza:
```
extend-commands:
    post-import:
      - exec: "drush vset -y site_name DEVDEV"
      - exec: "drush en -y environment_indicator"
```
(I was explicitly testing the validation on post-import to fail)